nodebuilder/p2p/config.go

```diff
@@ -92,3 +71,8 @@ func (cfg *Config) mutualPeers() (_ []peer.AddrInfo, err error) {
 
 	return peer.AddrInfosFromP2pAddrs(maddrs...)
 }
+
+// ValidateBasic performs basic validation of the config.
+func (cfg *Config) ValidateBasic() error {
+	return nil
```

Comment:
What about checking RefrefreshRoutingTablePeriod here instead of [here](https://github.com/celestiaorg/celestia-node/pull/997/files#diff-37ad66c6cbd50c5a4586f3613bfd1ac4ae6e7e744dda879ed5bb753da2b507f0R29)

Reply:
done